data-sources/firebase-data.js

```diff
@@ -80,38 +105,31 @@ function loginWithEmailPassword(_email: string, password: string): Promise {
 }
 
 /**
- *
- * @param emailAddress
+ * Resets a user's password
+ * @param {string} emailAddress
```

Comment:
This is standard JSDoc notation.  All public methods or exported functions should have this notation.   It's easy to do because most IDE's will auto-create stubs and warn you when they are missing.
 
VSCode plugin: 
https://marketplace.visualstudio.com/items?itemName=stevencl.addDocComments

I use Webstorm which does it automagically.